### DISCUSSION TITLE
评论功能不能在所有的博文中使用

### DISCUSSION THREAD

**Discussion by paneersblog** [walinejs/waline#815] [Q&A] [answered]
我的blog主题是fluid，评论功能只有在早期的几个页面中正常，后来hexo new "*"的都不能用了哎，请问这个是因为什么呀
附上链接 paneersblog.github.io
我偏向于小白一点...

**Comment by lizheming** [comment, marked as answer]
提供个具体不能用的页面地址看一下

**Comment by paneersblog** [comment]
![image](https://user-images.githubusercontent.com/98444325/153215669-2cac7233-b104-436b-a20c-7c4de83e54e3.png)
![image](https://user-images.githubusercontent.com/98444325/153215738-d67d36a9-cb5f-4e2d-b79d-d884b8484fe6.png)

第一张图是这个样子的 第二张图是可以用的

**Reply by paneersblog** [reply]
https://paneersblog.github.io/1.html  
这个链接就是图一

**Reply by paneersblog** [reply]
阿抱歉抱歉，我问题描述的有问题..是在手机端部分不可以（其实只有某几个可以），在电脑上是没问题的

**Reply by Mister-Hope** [reply]
我真心觉得可能是你手机端的网络问题。

![image](https://user-images.githubusercontent.com/33315834/153218862-0b353b6e-bee5-4edc-be42-36c26cb9db6f.png)

如果你用的主题对 waline 处理没有问题的话，可能是你的手机没有成功加载这个链接。

如果不是这个问题，那大概率可能是你所用的主题没有正确在手机端初始化 Waline。

**Reply by paneersblog** [reply]
喔，明白了！谢谢！！！
